plugins/okta/pkg/okta/okta.go

```diff
@@ -207,6 +214,8 @@ func (oktaPlugin *Plugin) Fields() []sdk.FieldEntry {
 		{Type: "string", Name: "okta.target.group.id", Desc: "Target Group ID"},
 		{Type: "string", Name: "okta.target.group.aternateid", Desc: "Target Group Alternate ID"},
 		{Type: "string", Name: "okta.target.group.name", Desc: "Target Group Name"},
+		{Type: "uint64", Name: "okta.mfa.failure.countlast", Desc: "Count of MFA failures in last seconds", Arg: sdk.FieldEntryArg{IsRequired: true, IsKey: true}},
+		{Type: "uint64", Name: "okta.mfa.deny.countlast", Desc: "Count of MFA denies in last seconds", Arg: sdk.FieldEntryArg{IsRequired: true, IsIndex: true}},
```

Comment:
The first field is `IsKey` but the latter is `IsIndex`

Is that perhaps a mistake?

Reply:
total mistake, thanks.